src/core/java/com/enderio/core/client/gui/widgets/EnumIconWidget.java

```diff
@@ -121,15 +121,15 @@ private static int getRow(int index) {
     private T tooltipDisplayCache;
 
     @Override
-    public void renderWidget(PoseStack pPoseStack, int pMouseX, int pMouseY, float pPartialTicks) {
-         if (isHovered && isActive()) {
-             addedOn.renderTooltipAfterEverything(pPoseStack, List.of(optionName, getter.get().getTooltip().copy().withStyle(ChatFormatting.GRAY)), pMouseX,
-                 pMouseY);
-         }
+    public void renderWidget(GuiGraphics guiGraphics, int pMouseX, int pMouseY, float pPartialTicks) {
+        if (isHovered && isActive()) {
+           addedOn.renderTooltipAfterEverything(guiGraphics, List.of(optionName, getter.get().getTooltip().copy().withStyle(ChatFormatting.GRAY)), pMouseX,
+                pMouseY);
```

Comment:
has this one parameter really be on it's own line because it's too long? maybe disable formatting before this line and enable after this one or extract the tooltipformatting